worker/src/main/scala/com/lucidchart/piezo/ConnectionProvider.scala

```diff
@@ -21,7 +21,7 @@ class ConnectionProvider(props: Properties) {
   val logger = LoggerFactory.getLogger(this.getClass)
   private val dataSource = props.getProperty("org.quartz.jobStore.dataSource")
   private val jdbcURL = if (dataSource != null) props.getProperty("org.quartz.dataSource." + dataSource + ".URL") else null
-  private val detectIpAddressFailover = if (dataSource != null) props.getProperty("org.quartz.dataSource." + dataSource + ".ipFailover") == "true" else false
+  private val detectIpAddressFailover = props.getProperty("supportIPFailover") == "true"
```

Comment:
I assume `getProperty` handles null values properly?

Reply:
Yes, `props.getProperty("supportIPFailover")` will return `null` if the value is not in the config. And `null == "true"` will return false